cpmpy/expressions/globalconstraints.py

```diff
@@ -157,13 +157,14 @@ def decompose_negation(self):
         from .python_builtins import all
         return [~all(self.decompose())]
 
-    def is_total(self):
```

Comment:
You can probably split off the removal of is_total, I'll happily do a swift review.